CHANGELOG.md

```diff
@@ -139,6 +139,10 @@
 * [#8146](https://github.com/rubocop-hq/rubocop/pull/8146): Use UTC in RuboCop todo file generation. ([@mauro-oto][])
 * [#8178](https://github.com/rubocop-hq/rubocop/pull/8178): Mark unsafe for `Lint/RaiseException`. ([@koic][])
 
+### Bug fixes
+
+* [#8102](https://github.com/rubocop-hq/rubocop/issues/8102): Consider class-length instead of block-length for `Struct.new`. ([@tejasbubane][])
```

Comment:
Can you move to the `## master (unreleased)` section.

Reply:
Done.